src/queries.js

```diff
@@ -44,9 +44,18 @@ export function appQuery(appbucket, limit) {
 export function browserSummaryQuery(browserbucket, windowbucket, afkbucket, count, filterAFK) {
   var browser_appnames = "";
   if (browserbucket.endsWith("-chrome")){
-    browser_appnames = JSON.stringify(["Google-chrome", "chrome.exe", "Chromium", "Google Chrome", "Chromium-browser", "Chromium-browser-chromium", "Google-chrome-beta", "Google-chrome-unstable"]);
+    browser_appnames = JSON.stringify([
+        "Google-chrome", "chrome.exe", "Google Chrome",
+        "Chromium", "Chromium-browser", "Chromium-browser-chromium",
+        "Google-chrome-beta", "Google-chrome-unstable",
+        // FIXME: Are these correct? Does the bucketname of opera and brave end with "-chrome"?
+        "opera.exe", "brave.exe",
```

Comment:
For Brave on macOS it is `Brave Browser`.

```
kerem             5202   0.0  1.9  5354244 315044   ??  S     5:13PM   2:38.60 /Users/kerem/Applications/Brave Browser.app/Contents/MacOS/Brave Browser
```

Reply:
For Vivaldi on macOS it's just `Vivaldi`. I downloaded Vivaldi browser just to check this out.

```
kerem            17897   0.0  0.9  4883304 145412   ??  S     5:51PM   0:06.62 /Users/kerem/Applications/Vivaldi.app/Contents/MacOS/Vivaldi -psn_0_8038314
```

Reply:
For Opera on macOS it is `Opera`.

```
kerem            22320   0.0  1.0  5345560 162692   ??  S     6:01PM   0:03.83 /Users/kerem/Applications/Opera.app/Contents/MacOS/Opera -psn_0_9988486
```